apps/sms/lib/bridge/bridge.js

```diff
@@ -32,13 +32,27 @@ module.exports = Client;
 /**
```

Comment:
Ah, sorry :) Yeah, once we move ActivityShim to iframe/hidden window we'll need only client.js, you're right :)